src/python/tensorflow_cloud/core/validate.py

```diff
@@ -156,6 +163,13 @@ def _validate_cluster_config(chief_config, worker_count, worker_config):
             "Received {}.".format(worker_count)
         )
 
+    if (
```

Comment:
Please update to:
if machine_config.is_tpu_config(worker_config):
   if worker_count > 1:
      raise ...

   if version >= ...:
      raise ...

Reply:
Done with a minor change `if "worker_count > 1:"` to `if "worker_count != 1:"` since worker_count=0 is not a valid input. could you confirm this is a correct undrestanding?

Reply:
Yeah that's fine.